Assets/Mirage/Runtime/Sockets/Udp/UdpSocketFactory.cs

```diff
@@ -2,40 +2,85 @@
 using System.Net;
 using System.Net.Sockets;
 using Mirage.SocketLayer;
+using NanoSockets;
 using UnityEngine;
 
 namespace Mirage.Sockets.Udp
 {
+    public enum SocketLib { Automatic, NanoSockets, Net };
```

Comment:
Maybe rename NanoSockets to Native and Net to Managed? Might be easier to understand

Reply:
Done